CHANGELOG.md

```diff
@@ -0,0 +1,40 @@
+
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [1.0.0] - 2023-01-10
+
+### Added
+
```

Comment:
```suggestion

### Changed
* Use Kafka clients version 3.3.1.
* Refresh dependencies versions.

### Deprecated
* Deprecate checksum from ConsumerRecord (deprecated since client version 3.0)
```

Reply:
done